src/components/views/elements/LabelledToggleSwitch.js

```diff
@@ -31,15 +31,29 @@ export default class LabelledToggleSwitch extends React.Component {
 
         // Whether or not to disable the toggle switch
         disabled: PropTypes.bool,
+
+        // True to put the toggle in front of the label
+        // Default false.
+        toggleInFront: PropTypes.bool,
```

Comment:
Why do we sometimes want the opposite order?

Reply:
Usually the toggle is at the end of the label, however in a dialog this doesn't make much sense to the user. This is the first instance of needing to do this, and it seems easiest to just put a boolean on it.


Everywhere else:
![image](https://user-images.githubusercontent.com/1190097/54875358-35516900-4dc3-11e9-93f7-bc16dfd92b9a.png)

Widget dialog:
![image](https://user-images.githubusercontent.com/1190097/54875355-25398980-4dc3-11e9-8da9-18839909127c.png)

Reply:
Hmm, well, technically it's fine. I'd suggest asking Nad if he would prefer only having one layout of toggles or if he agrees it seems better reversed as you've done it here.

Reply:
Summarizing the discussion from earlier today: We'll take care of this in https://github.com/vector-im/riot-web/issues/7566